src/validator.directive.js

```diff
@@ -225,6 +228,7 @@
             ctrl.$render();
             if (attrs.messageId) angular.element(document.querySelector('#' + attrs.messageId)).html('');
             else element.next().html('');
+            if ($validationProvider.validCallback) $validationProvider.validCallback(element);
```

Comment:
What if we have the reset callback?
https://github.com/huei90/angular-validation/pull/172